server/api/dev.js

```diff
@@ -0,0 +1,214 @@
+/*
+  Easier to refresh entire dev entry instead of upsert/update the dates
+  ??? What is the removed column for in entries?
+*/
+
+const db = require('../db');
+const { createLogger } = require('../logger');
+
+const NOW = new Date();
+const VOTE_MAX = 5;
+const BACKGROUND_COLORS = ['#000', '#FFF', '#4b91e3'];
+
+const logger = createLogger('API/DEV');
+
+const getDevContest = (status) => {
+  const contestStatuses = ['prerelease', 'reset', 'submission', 'review', 'voting', 'results'];
+  const contestIndex = contestStatuses.indexOf(status);
+  const isAfter = (lastStatus) => contestIndex > contestStatuses.indexOf(lastStatus);
+
+  const yearEnd = NOW.getMonth() === 11; // getMonth is zero-based
```

Comment:
FYI, there are actually two contests in December, a regular one and a year-one end. Not sure if this affects anything here, though.

Reply:
This should not impact the `dev` contest